torchci/test/mergeCommands.test.ts

```diff
@@ -296,6 +303,13 @@ describe("merge-bot", () => {
     const pr_number = event.payload.issue.number;
     const comment_number = event.payload.comment.id;
     const scope = nock("https://api.github.com")
+      .post(`/repos/${owner}/${repo}/issues/${pr_number}/labels`, (body) => {
+        expect(JSON.stringify(body)).toContain(
+          `"labels":["ciflow/trunk"]`
+        );
+        return true;
```

Comment:
nit: even though we do expect all test to send this label, we don't really need to enforce that every test actually does set that label. A single test asserting that expectation would be sufficient, and would make the test suite be less brittle to future changes

Reply:
We actually need to do this or nock will complain